### DISCUSSION TITLE
kraken现在支持flutter2.0吗？

### DISCUSSION THREAD

**Discussion by neterji** [openkraken/kraken#173] [Q&A]
kraken现在支持flutter2.0吗？

**Comment by andycall** [comment]
还没有支持，目前升级到 flutter 2.0 需要先统一升级到 dart null safety，预计在 0.8 版本 （计划是5-6月）升级

**Reply by neterji** [reply]
也就是说5-6月份可以使用flutter2.0了呗

**Reply by neterji** [reply]
另外css风格支持有时间点吗？

**Reply by andycall** [reply]
CSS 语法的支持要到 0.9 版本的发布了 （8-9月）

**Reply by andycall** [reply]
> 也就是说5-6月份可以使用flutter2.0了呗

是的，支持 flutter 2.0 很容易
